content/200-orm/050-overview/500-databases/850-planetscale.mdx

```diff
@@ -63,9 +62,15 @@ Every PlanetScale database is created with a branch called `main`, which is init
 
 If you try to push to a production branch, you will get the [error message](/orm/reference/error-reference#p3022) `Direct execution of DDL (Data Definition Language) SQL statements is disabled on this database.`
 
-## How to emulate relations in Prisma Client
+## How to use foreign key constraints with PlanetScale and Prisma ORM
```

Comment:
"use foreign key constraints" is not really correct. "foreign key constraints" are things on the database side. By default you can not use those with Planetscale, and instead emulate the _results_ of setting a foreign key constraint on the Prisma side.

Maybe just "How to use relations with PlanetScale"? Because you either do that with `relationMode=prisma` or changing their setting.

Reply:
Hmm I guess that's fair... wdyt about an additional note about "referential integrity", e.g. "How to use relations (and enable referential integrity) with PlanetScale"? does this add value or should we go with your shorter suggestion?

```suggestion
## How to use relations (and enable referential integrity) with PlanetScale
```

Reply:
Both works for me. Your suggestion is both Prisma and database level, that is pretty good 👍